inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/model/KnowledgeBase.java

```diff
@@ -382,7 +381,13 @@ public void applyMapping(KnowledgeBaseMapping aMapping)
         setPropertyTypeIri(aMapping.getPropertyTypeIri());
         setPropertyLabelIri(aMapping.getPropertyLabelIri());
         setPropertyDescriptionIri(aMapping.getPropertyDescriptionIri());
-        applyRootConcept(aMapping);
+        if (explicitlyDefinedRootConcepts.isEmpty()) {
+            ValueFactory vf = SimpleValueFactory.getInstance();
+            for (String rootConcept : aMapping.getRootConcepts()) {
+                explicitlyDefinedRootConcepts.add(vf.createIRI(rootConcept));
+            }
+        }
+        setExplicitlyDefinedRootConcepts(explicitlyDefinedRootConcepts);
```

Comment:
`setExplicitlyDefinedRootConcepts(explicitlyDefinedRootConcepts);` translates to `explicitlyDefinedRootConcepts  = explicitlyDefinedRootConcepts`, right? So it's a no-operation?